### PR TITLE
Update directory tree of output segments from VoD pipeline

### PR DESCRIPTION
**Pull request by emranemran**
Fix https://github.com/livepeer/catalyst-api/issues/134

### CODE REVIEW

**Comment by emranemran** [comment]
TODOs before merging: 
* Fix failing tests
* Update rendition folder names to use profile names instead
* Double check re-use of any unnecessary variables introduced in path/url manipulation

**Review by thomshutt — APPROVED** [review]
LGTM pending the changes you mentioned

**Comment by emranemran** [comment]
These changes are being handled in a follow-up PR as it required refactor of larger parts for code readability: 
* Update rendition folder names to use profile names instead (example 360p/ instead of rendition-1/
* Update profile list to use 240p, 480p, 720p

**Pull request merged**
